src/store/features/featuresSlice.ts

```diff
@@ -28,6 +29,7 @@ const featuresSlice = createSlice({
       state.discoveryEnabled = payload.discoveryEnabled
       state.checklistEnabled = payload.checklistEnabled
       state.followHotspotEnabled = payload.followHotspotEnabled
+      state.qrOnboardEnabled = payload.qrOnboardEnabled
```

Comment:
Should this be false by default? Did we reuse the list from the last test group?

Reply:
It's a different test group. We want it to be undefined at first so that if someone scans a QR code from the system camera we know to set the link to `unhandled` until the API responds with the actual state of `qrOnboardEnabled`.